target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -131,15 +131,6 @@ mediatek_setup_macs()
 	bananapi,bpi-r4)
 		wan_mac=$(macaddr_add $(cat /sys/class/net/eth0/address) 1)
 		;;
-	cmcc,rax3000m)
-		case "$(cmdline_get_var root)" in
-		/dev/mmc*)
-			wan_mac=$(mmc_get_mac_binary factory 0x2a)
-			lan_mac=$(mmc_get_mac_binary factory 0x24)
-			label_mac=$wan_mac
-		;;
-		esac
-		;;
```

Comment:
why remove this? emmc mac need to be fixup
and as what I can see, label_mac is `factory 0x24`, not the 0x2a
@1715173329

Reply:
Also on nand, label_mac is `factory 0x24`

Reply:
> why remove this? emmc mac need to be fixup

Instead of fixing it in userspace we can use NVMEM framework to let kernel read the address, see

https://github.com/openwrt/openwrt/pull/15054/commits/d7e2f2c89e3697d109e4b55587aaa836fd66a385#diff-84c89974add3e01d861050f6a1205ca166b10667265e16e0dc5440009a2dabacR59

Reply:
okay, so we have to add `label-mac-device` aliases to correct the label-mac?
```
         aliases {
                label-mac-device = &gmac1;
        };
```

Reply:
`label-mac-device` seems a optional binding. Do you have any application rely on it?

Reply:
> Also on nand, label_mac is `factory 0x24`

True issue is the assigned mac address was reversed.